tests/unit_tests/test_unit.py

```
@@ -781,239 +785,208 @@ def test_length_restrictions(tiny_fasta_file):
-        missed_cleavages=0,
-        min_peptide_len=6,
-        max_peptide_len=50,
-        max_mods=0,
-        precursor_tolerance=20,
-        isotope_error=[0, 0],
-        allowed_fixed_mods="C:C+57.021",
-        allowed_var_mods=(
-            "M:M+15.995,N:N+0.984,Q:Q+0.984,"
-            "nterm:+42.011,nterm:+43.006,nterm:-17.027,nterm:+43.006-17.027"
+@pytest.mark.parametrize(
+    "enzyme,digestion,expected",
```

Ruff (0.15.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)